ui/desktop/app/controllers/scopes/scope/projects/targets/target.js

```diff
@@ -13,6 +13,7 @@ export default class ScopesScopeProjectsTargetsTargetController extends Controll
   queryParams = [{ isConnecting: { type: 'boolean' } }];
 
   @tracked isConnecting = false;
+  @tracked connectError;
```

Comment:
If we are now using a `connectError`, do we still need `isConnectionError` in the route?

Reply:
None blocking comment: Indeed with @ZedLi comment, in case we do not need it, I am fine getting that change on this PR or on a separate as tech debt (for pragmatism sake and promote staging ASAP).